frontend/src/routes/+page.svelte

```diff
@@ -1,2 +1,194 @@
+                });
+                timeline.from(splitText.chars, {
+                    y: 10,
+                    opacity: 0,
+                    duration: animationDuration,
+                    stagger: 0.025
+                });
+            });
+        }
+        if (animatePopIn) {
+            timeline.fromTo(
+                animatePopIn,
+                {
+                    opacity: 0,
+                    scale: 0.1
+                },
+                {
+                    opacity: 1,
+                    scale: 1,
+                    duration: animationDuration * 0.5,
+                    stagger: 0.05
+                }
+            );
+        }
+    }
+
+    /* ---- ANIMATION FOR PARTICLES ---- */
+    const randomSpringCoords = () => {
+        const xMin = 10;
+        const xMax = 40;
+        const yMin = 0;
+        const yMax = 90;
+        const coords = spring({
+            x: Math.random() * (xMax - xMin) + xMin,
+            y: Math.random() * (yMax - yMin) + yMin
+        });
+        coords.stiffness = 0.002;
+        coords.damping = 1;
+        const setNewCoords = () => {
+            coords.set({
+                x: Math.random() * (xMax - xMin) + xMin,
+                y: Math.random() * (yMax - yMin) + yMin
+            });
+        };
+        setNewCoords();
+        setInterval(setNewCoords, 10000);
+        return coords;
+    };
+
+    const atomCoords = randomSpringCoords();
+    const molStructureCoords = randomSpringCoords();
+    const molCoords = randomSpringCoords();
+    const roundBeakerCoords = randomSpringCoords();
+</script>
+
+<svelte:head>
+    <title>SkoutaLab</title>
+</svelte:head>
+
+<div
+    class={`relative z-0 flex items-center justify-around lg:h-[calc(100vh-60px)] dark:bg-slate-800 text-slate-800 dark:text-white`}
+>
+    <div id="hero" class="z-0">
+        <div bind:this={animateTextWrapper}>
+            <h1 class="mb-4 text-8xl font-bold text-green-600 dark:text-green-300">SkoutaLab</h1>
+            <p class="mb-8 text-2xl font-semibold">
+                Organic Chemistry, but now more fun and interactive!
+            </p>
+        </div>
+        <button
+            bind:this={animatePopIn}
+            class="relative rounded-xl bg-neutral-900 px-8 py-2 text-2xl font-semibold text-white opacity-0 transition-all duration-300 hover:cursor-pointer hover:bg-neutral-700 dark:bg-slate-200 dark:text-slate-800 dark:hover:bg-slate-400"
+            on:focus={() => goto('/login')}
+        >
+            Start learning
+        </button>
+    </div>
+    <ul id="hover-card" class="relative flex w-96 flex-col">
+        <li class="group relative flex aspect-square w-1/4 items-center justify-center self-center hover:cursor-pointer">
+            <div
+                class="absolute top-0 -z-10 aspect-square w-full rotate-45 rounded-lg bg-slate-200 dark:bg-slate-400 transition-[width,top] group-hover:top-6 group-hover:z-10 group-hover:w-[225%]"
```

Comment:
There are a lot of duplicated css properties here in line 127, 141, 152, 166 such as `absolute -z-10 aspect-square w-full rotate-45 rounded-lg bg-slate-200 dark:bg-slate-400 group-hover:top-6 group-hover:z-10 group-hover:w-[225%]`. I'd suggest to declare and use a common css class, that way it is easier for the future reader understands what is going on.

Reply:
Got it will change that later